doc.md

```diff
@@ -0,0 +1,7 @@
+| Method     |    Endpoint   | possible Query Params             |
```

Comment:
Having this habit or organizing methods and endpoints is going to be useful later on. Good job.